# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/systemservices/objc_video_capture/video_capture_avfoundation.mm
# Commit: 731dcbc83284
# Full Hash: 731dcbc83284d24f6177c1917ee6e1b17a44bbdc
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2023-01-13 21:39:47
# Regressor Bug: 1806605
# File Overlap Count: 1
# Description:
#   Bug 1806605 - Fix PerformanceRecorder not adding markers in VideoCaptureAvFoundation. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D165823
# ==============================================================================

diff -r 39508d8be196 -r 731dcbc83284 dom/media/systemservices/objc_video_capture/video_capture_avfoundation.mm
--- a/dom/media/systemservices/objc_video_capture/video_capture_avfoundation.mm	Fri Jan 13 14:24:59 2023 +0000
+++ b/dom/media/systemservices/objc_video_capture/video_capture_avfoundation.mm	Fri Jan 13 14:25:00 2023 +0000
@@ -242,7 +242,7 @@
 void VideoCaptureAvFoundation::StartFrameRecording(int32_t aWidth, int32_t aHeight) {
   MaybeRegisterCallbackThread();
   MutexLock lock(&api_lock_);
-  if (MOZ_UNLIKELY(mTrackingId)) {
+  if (MOZ_UNLIKELY(!mTrackingId)) {
     return;
   }
   auto fromWebrtcVideoType = [](webrtc::VideoType aType) -> CaptureStage::ImageType {
